doc/install/install-ceph-gateway.rst

```diff
@@ -2,227 +2,113 @@
+   networks, and SELinux should be in permissive mode in rpm-based distros.
 
+The :term:`Ceph Object Gateway` daemon runs on Apache and FastCGI.
 
-RPM Packages
-------------
-
-To install Apache with 100-continue, execute the following steps:
-
-#. Install ``yum-plugin-priorities``. ::
-
-	sudo yum install yum-plugin-priorities
-
-#. Ensure ``/etc/yum/pluginconf.d/priorities.conf`` exists.
-
-#. Ensure ``priorities.conf`` enables the plugin. :: 
-
-	[main]
-	enabled = 1
-
-#. Add a ``ceph-apache.repo`` file to ``/etc/yum.repos.d``. Replace 
-   ``{distro}`` with the name of your distribution (e.g., ``centos6``, 
-   ``rhel6``, etc.) ::
-
-	[apache2-ceph-noarch]
-	name=Apache noarch packages for Ceph
-	baseurl=http://gitbuilder.ceph.com/apache2-rpm-{distro}-x86_64-basic/ref/master
-	enabled=1
-	priority=2
-	gpgcheck=1
-	type=rpm-md
-	gpgkey=https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/autobuild.asc
-
-	[apache2-ceph-source]
-	name=Apache source packages for Ceph
-	baseurl=http://gitbuilder.ceph.com/apache2-rpm-{distro}-x86_64-basic/ref/master
-	enabled=0
-	priority=2
-	gpgcheck=1
-	type=rpm-md
-	gpgkey=https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/autobuild.asc
+To run a :term:`Ceph Object Storage` service, you must install Apache and
+Ceph Object Gateway daemon on the host that is going to provide the gateway
+service, i.e, the ``gateway host``. If you plan to run a Ceph Object Storage
+service with a federated architecture (multiple regions and zones), you must
+also install the synchronization agent.
 
+.. note:: Previous versions of Inktank Ceph Enterprise shipped with
+   ``mod_fastcgi``. The current version ships with ``mod_proxy_fcgi`` instead.
 
-#. Add a ``ceph-fastcgi.repo`` file to ``/etc/yum.repos.d``. Replace 
-   ``{distro}`` with the name of your distribution (e.g., ``centos6``, 
-   ``rhel6``, etc.) ::
+In distros that ship Apache 2.4 (such as RHEL 7, CentOS 7 or Ubuntu 14.04
+``Trusty``), ``mod_proxy_fcgi`` is already present. When you install the
+``httpd`` package with ``yum``, ``mod_proxy_fcgi`` becomes available for use
+on your server.
 
-	[fastcgi-ceph-basearch]
-	name=FastCGI basearch packages for Ceph
-	baseurl=http://gitbuilder.ceph.com/mod_fastcgi-rpm-{distro}-x86_64-basic/ref/master
-	enabled=1
-	priority=2
-	gpgcheck=1
-	type=rpm-md
-	gpgkey=https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/autobuild.asc
-	
-	[fastcgi-ceph-noarch]
-	name=FastCGI noarch packages for Ceph
-	baseurl=http://gitbuilder.ceph.com/mod_fastcgi-rpm-{distro}-x86_64-basic/ref/master
-	enabled=1
-	priority=2
-	gpgcheck=1
-	type=rpm-md
-	gpgkey=https://ceph.com/git/?p=ceph.git;a=blob_plain;f=keys/autobuild.asc
+In distros that ship Apache 2.2 (such as RHEL 6, CentOS 6 or Ubuntu 12.04
+``Precise``), ``mod_proxy_fcgi`` comes as a separate package. In
+**RHEL 6/CentOS 6**, it is available in ``EPEL 6`` repo and can be installed with
+``yum install mod_proxy_fcgi``. For **Ubuntu 12.04**, a backport has been created
```

Comment:
Let's change the Ubuntu 12.04 section here to say "For **Ubuntu 12.04**, a backport is in progress" and link to https://bugs.launchpad.net/precise-backports/+bug/1422417